apps/dashboard/shared/dao-config/gtc.ts

```diff
@@ -36,15 +36,13 @@ export const GTC: DaoConfiguration = {
       proposalThreshold: "150k GTC",
     },
   },
-  // attackProfitability: {
-  //   riskLevel: RiskLevel.HIGH,
-  //   supportsLiquidTreasuryCall: false,
-  //   attackCostBarChart: {
-  //     OptimismTimelock: "",
-  //     OptimismTokenDistributor: "",
-  //     OptimismUniv3Uni: "",
-  //   },
-  // },
+  attackProfitability: {
+    riskLevel: RiskLevel.HIGH,
+    supportsLiquidTreasuryCall: true,
+    attackCostBarChart: {
+      GitcoinTimelock: "0x57a8865cfB1eCEf7253c27da6B4BC3dAEE5Be518",
```

Comment:
could you please verify with @Zeugh-eth whether we should add [these address](https://github.com/blockful/anticapture/blob/main/apps/indexer/src/lib/constants.ts#L186) on the frontend as DAO owner addresses, or remove them from the treasury indexing